worlds/alttp/Options.py

```diff
@@ -722,6 +722,15 @@ class AllowCollect(Toggle):
     display_name = "Allow Collection of checks for other players"
 
 
+class Removed(FreeText):
+    default = ""
+    visibility = Visibility.none
+
+    def __init__(self, value: str):
+        if value:
+            raise Exception("Option removed, please update your options file.")
```

Comment:
This should probably be more specific about which player's yaml and which option?

Reply:
The option exception is caught and this information is appended to the trace. Admittedly, the webhost does only display the innermost exception, but in my opinion that's an unrelated problem that needs fixing seperately.